package-overrides/npm/angular-aria@1.5.0.json

```diff
@@ -0,0 +1,15 @@
+{
+  "format": "global",
+  "jspmNodeConversion": false,
+  "main": "angular-aria",
+  "dependencies": {
+    "angular": "npm:angular@^1.5.0"
```

Comment:
The issue here is that the ecosystem is currently still stuck on loading Angular from github. It may be best to just use the GitHub version of Angular for all of these to avoid compatibility issues.